### PR TITLE
OOP drill

### PR DESCRIPTION
**Pull request by gilRoz23**
wasnt at wednesday
I'll be glad for a review

### CODE REVIEW

File: src/main/kotlin/com/hibob/kotlinEx/Location.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.hibob.kotlinEx
```
Comment (liorsiv973) [inline src/main/kotlin/com/hibob/kotlinEx/Location.kt:1]: nice! if you would do interface instead would it make more sense?

File: src/main/kotlin/com/hibob/kotlinEx/Meeting.kt [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package com.hibob.kotlinEx
+
+open class Meeting (val name: String, val location: Location, val participants: List<Participant>){
+
+    open fun addParticipant(participant: Participant){
```
Comment (liorsiv973) [inline src/main/kotlin/com/hibob/kotlinEx/Meeting.kt:5]: both here and in addReviewer adding this won't change the list from outside.

File: src/main/kotlin/com/hibob/kotlinEx/kotlinBasics.kt [hunk 3]
```diff
@@ -1,31 +0,0 @@
-package com.hibob.kotlinEx
```
Comment (liorsiv973) [inline src/main/kotlin/com/hibob/kotlinEx/kotlinBasics.kt:1]: why is that here?

File: src/main/kotlin/com/hibob/kotlinEx/Location.kt (hunk 1 above)
Comment (gilRoz23) [reply src/main/kotlin/com/hibob/kotlinEx/Location.kt:1]: thank you for the advice! changed it

File: src/main/kotlin/com/hibob/kotlinEx/kotlinBasics.kt (hunk 3 above)
Comment (gilRoz23) [reply src/main/kotlin/com/hibob/kotlinEx/kotlinBasics.kt:1]: merged from the master

File: src/main/kotlin/com/hibob/kotlinEx/Location.kt [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+interface Location {
```
Comment (liorsiv973) [inline src/main/kotlin/com/hibob/kotlinEx/Location.kt:3]: looks good!!

File: src/main/kotlin/com/hibob/kotlinEx/kotlinBasics.kt (hunk 3 above)
Comment (liorsiv973) [reply src/main/kotlin/com/hibob/kotlinEx/kotlinBasics.kt:1]: so please remove it. it has nothing to do with this branch. try to rebase branch from master

**Review by liorsiv973 — APPROVED** [review]
